models/pathology_tumor_detection/configs/train.json

```diff
@@ -372,8 +372,7 @@
         }
     },
     "initialize": [
-        "$monai.utils.set_determinism(seed=15)",
-        "$setattr(torch.backends.cudnn, 'benchmark', True)"
+        "$monai.utils.set_determinism(seed=123)"
```

Comment:
Hi @drbeh , could you also update the weights and figures trained from deterministic settings?

Reply:
Sure, but it will take a while as we are updating the pre-trained weights to nv-imagenet too.

Reply:
I have updated the weights and figure for the deterministic training. We have postponed training with local weights, so maybe we can merge this till we have update with the nv-imagenet. Thanks